copyFile.test.js

```diff
@@ -1,8 +1,22 @@
 const fs = require('fs').promises;
-const copier = require('./copyFile');
+const { copyAFile } = require('./copyFile');
+
+describe('copyAFile fn', () => {
+    afterEach(() => {
+        return fs.rm('./newFile-copy.txt');
+    });
 
-describe('copier fn', () => {
     it('reads a file and writes a copy file', async () => {
 
+        return copyAFile('./newFile.txt', './newFile-copy.txt')
+            .then(() => {
+                return fs.readFile('./newFile.txt', 'utf-8');
+            })
+            .then(
+                newFile => {
+                    expect(newFile)
+                        .toEqual('I\'m not sure about this')
```

Comment:
super close, the test isn't passing for me right now bc it says that you are receiving "I am not sure about this" instead of 'I\'m not sure about this'. Super easy fix it seems, resubmit when you can!